### ISSUE TITLE
Clear local development with easy setup instructions. 

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#663]
**TL;DR**: There should be well flushed out local development install instructions

**Narrative**
Developers need clear documentation on how to setup and run Lookit on their local computer.  This would facilitate new developers and short term developers.  Bugs can arise from from have disparate environment setups.  An example of this would be newer versions of Postgres will run SQL differently than older ones causing performance issues.  

**Acceptance Criteria**
- All software versions should be the same across all development machines.  
- Python libraries should be hashed.  (I usually recommend against pinning Python Lib version, but we can talk about this.)

- [x] Tests
- [X] Scoping
- [X] Alternatives Considered
- [X] Rico sign-off

**Implementation Notes**

This is purely optional, and maybe I can do a short presentation on the benefits on any of these.  I would like to replace pip with poetry.  Additionally, we could either have a docker compose file that will run a Postgres DB or something more like asdf-vm.  

I'd love to hear about additional bits of software we can use to make our lives easier.
